relentless/simulate/lammps.py

```diff
@@ -321,14 +322,24 @@ def pair_map(sim,pair):
                     f = sim.potentials.pair.force((i,j))[flags]
                     for idx in range(Nr):
                         fw.write('{idx} {r} {u} {f}\n'.format(idx=idx+1,r=r[idx],u=u[idx],f=f[idx]))
+                        # find the largest r with a nonzero potential
+                        if abs(u[idx])>0.0 and u[idx+1]==0.0:
+                            rcut_val = r[idx]
+
+                    key = '{id_i},{id_j}'.format(id_i=id_i,id_j=id_j)
+                    rcut[key] = rcut_val
```

Comment:
Just key it on `i,j` since below you are going to do the same loop
```suggestion
                    rcut[(i,j)] = rcut_val
```